server/server.js

```diff
@@ -116,6 +116,11 @@ app.use('/config.js', (req, res, next) => {
 
 // Right the body to the req object. Fixes the issues body-parser causes for the proxies
 const onProxyReq = (proxyReq, req, res) => {
+  // Use the configured pulsar token to access pulsar services.
+  proxyReq.setHeader('Authorization', 'Bearer ' + cfg.globalConf.server_config.admin_token)
```

Comment:
How were we adding this token before?

Reply:
The token is passed in by the client, the problem is that we were sometimes seeing 401s due to requests being made before the page was fully loaded.  I thought it would be better to just attach the token for proxied requests which means that the token doesn't need to be passed to the browser for these requests.

Reply:
Ok, I removed it.